blackbox/docs/administration/hba.txt

```diff
@@ -119,9 +119,9 @@ using the ``trust`` method.
 that the user ``dustin`` can authenticate to CrateDB over HTTP protocol from
 the ``172.16.0.0`` IP Address using the ``trust`` method.
 
-``{user: trinity, address: 127.0.0.1, protocol: pg, ssl: off}`` means that the
-user ``trinity`` can authenticate to CrateDB over Postgres Wire Protocol from
-the ``127.0.0.1`` IP Address only if no SSL/TLS connection is used. Since no
+``{user: trinity, address: 127.0.0.1, protocol: http, ssl: off}`` means that
```

Comment:
the example above still says protocol: pg, please change
https://github.com/crate/crate/blob/8d286dfa88b2d3ac5e30fd70a8c3e6fb708167e6/blackbox/docs/administration/hba.txt#L81